i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx

```diff
@@ -0,0 +1,215 @@
+---
+sidebar_position: 2
+---
+
+# Setup & Configuration
+
+## Configuration
+
+### A note on individual tastes
+
+::::caution
+evcc has a lot of settings, knobs, and dials. The default parameters are based on lots of experience from many users, and most of the time, the default should work well for you.
+
+**If you do start making changes, remember the golden rule:**
+
+:::danger
+**Since parameters sometimes influence each other, it is STRONGLY encouraged to only make changes to one setting at at time, and test the effects of each change before making further modifications.**
+:::
```

Comment:
I wont do the nested blocks here. I think CAUTION+DANGER is a little much :D

Reply:
Think I agree here, but I wanna check on a laptop to see what looks best, gimme a bit 👀

Reply:
Resolved in [af64646](https://github.com/evcc-io/docs/pull/426/commits/af646464ccfae587ce7dffba35428f2c5138968e)